infrastructure/critical/modules/rds/security_groups.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_security_group" "database_sg" {
+  description = "controls direct access to application instances"
+  vpc_id      = var.vpc_id
+  name        = "${var.database_name}_sg"
+
+  ingress {
+    protocol  = "tcp"
+    from_port = 3306
+    to_port   = 3306
+
+    cidr_blocks = [
+      var.admin_cidr_ingress,
+    ]
+  }
```

Comment:
Do we need to be able to access the database directly?

Reply:
Assuming for debugging or something. This I might not change given it's the ID minter